### PR TITLE
UE-239 Renamed all instances of 'stanford_starter'

### PR DESCRIPTION
**Pull request by jdwjdwjdw**
# READY FOR REVIEW

# Summary
- Renamed all instances of 'stanford_starter'
- Updated README.md and the theme info file
- Built files

# Review By (Date)
- When convenient

# Steps to Test
1. Review code.

# Affected Projects or Products
- VPUE Undergrad

# Associated Issues and/or People
[UE-239](https://stanfordits.atlassian.net/browse/UE-239)
[UE-240](https://stanfordits.atlassian.net/browse/UE-240)
[UE-242](https://stanfordits.atlassian.net/browse/UE-242)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# [Stanford Starter](https://github.com/SU-SWS/stanford_starter)
+# [vpue_undergrad_subtheme](https://github.com/SU-SWS/vpue_undergrad_subtheme)
```
Comment (buttonwillowsix) [inline README.md]: This is, I think, the pretty name we will see in the UI. Let's make this just say "VPUE Undergrad"

**Review by buttonwillowsix — CHANGES_REQUESTED** [review]
This looks good with one minor, minor tweak! See comments. Awesome!

File: README.md (hunk 1 above)
Comment (jdwjdwjdw) [reply README.md]: Agreed - that looks much nicer in the UI

**Review by buttonwillowsix — APPROVED** [review]

**Comment by sherakama** [comment]
You're #1!!!!

**Pull request merged**